src/main/java/nixy/Nixy.java

```diff
@@ -0,0 +1,125 @@
+package nixy;
+
+import nixy.exceptions.NixyException;
+import nixy.parse.Parsed;
+import nixy.parse.Parser;
+import nixy.task.Task;
+import nixy.task.TaskList;
+import nixy.ui.Ui;
+
+/**
```

Comment:
I like the constant use of javadocs to tell users what the file and functions represent!